core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -49,6 +49,17 @@ oppia.factory('PlaythroughService', [
     var cycleIdentifier = {};
     var visitedStates = [];
 
+    /**
+     * A quit action is recorded every time a MaybeQuit event is encountered.
+     * This function extracts the old quit action out of the playthrough
+     * actions list.
+     */
+    var removeOldQuitAction = function() {
```

Comment:
`removeOldQuitActions` since there may be many.

Reply:
Nope. there won't be many. Whenever a new quilt action is introduced, the old quit action is removed first. So at any point, there will only be a maximum of two quit actions.